decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb

```diff
@@ -0,0 +1,143 @@
+                presented_author.profile_path,
+                target: :blank
+              ) %>
+              <% if presented_author.can_be_contacted? && presented_author.nickname != present(current_user).nickname %>
+                <%= icon_link_to "envelope-closed", current_or_new_conversation_path_with(presented_author), t("decidim.contact"), class:"action-icon--new" %>
+              <% end %>
+            </li>
+          <% end %>
+        </ul>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".body" %>:</strong>  <%= simple_format(present(proposal).body(strip_tags: true)) %>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".created_at" %>:</strong> <%= l proposal.created_at, format: :decidim_short %>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".votes_count" %>:</strong> <%= proposal.proposal_votes_count %>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".votes_ranking" %>:</strong> <%= i18n_votes_ranking_for(proposal) %>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".endorsements_count" %>:</strong> <%= proposal.proposal_endorsements_count %>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".endorsements_ranking" %>:</strong> <%= i18n_endorsements_ranking_for(proposal) %>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".amendments_count" %>:</strong> <%= proposal.amendments.count %>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".comments_count" %>:</strong> <%= proposal.comments.count %>
+      <ul id="proposal-comments-alignment-count">
+        <li><strong><%= t ".comments_positive_count" %>:</strong> <%= proposal.comments.positive.count %></li>
+        <li><strong><%= t ".comments_neutral_count" %>:</strong> <%= proposal.comments.neutral.count %></li>
+        <li><strong><%= t ".comments_negative_count" %>:</strong> <%= proposal.comments.negative.count %></li>
+      </ul>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".endorsers" %>:</strong>
+        <ul id="proposal-endorsers-list">
+          <% presented_endorsers = endorsers_presenters_for(proposal).to_a %>
+          <% presented_endorsers.first(5).each do |presented_endorser| %>
+            <li>
+              <%= link_to_if(
+                presented_endorser.profile_path.present?,
+                presented_endorser.name,
+                presented_endorser.profile_path,
+                target: :blank
+              ) %>
+            </li>
+          <% end %>
+          <% if presented_endorsers.count > 5 %>
+            <li>
+              <%= link_to(
+                  t(".n_more_endorsers", count: presented_endorsers.count - 5),
+                  resource_locator(proposal).path
+              ) %>
+            </li>
+          <% end %>
+        </ul>
+    </div>
+
+    <div class="row column">
+      <strong><%= t ".related_meetings" %>:</strong>
+        <ul id="related-meetings">
```

Comment:
When linking a meeting to a proposal
![imatge](https://user-images.githubusercontent.com/199462/74109543-9a04e600-4b84-11ea-831c-8f325ae94681.png)

I can't see the meeting listed:
![imatge](https://user-images.githubusercontent.com/199462/74109556-b86ae180-4b84-11ea-93f6-92f87968aca8.png)

Can it be something related to my local environment?

Reply:
mmm.. I think this is because of this: https://github.com/decidim/decidim/pull/5674
so, ok

Reply:
That's correct, it's due to #5674!